spec/controllers/families_controller_spec.rb

```diff
@@ -40,9 +43,9 @@
         delete :destroy, params: { id: family.id }
 
         expect(flash[:notice])
-        .to match(
-          I18n.t("flash.actions.destroy.notice", resource_name: "Family")
-        )
+          .to match(
```

Comment:
Again, `.` should be in the end of previous line.
But a suggestion to assign the expected to a variable. Something like:
```
notice = I18n.t("flash.actions.destroy.notice", resource_name: "Family")

delete :destroy, params: { id: family.id }

expect(flash[:notice]).to match(notice)
```

Reply:
Done in 25ddc19